### PR TITLE
feat(commons): Add handle validation

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: packages/commons/src/main/util/ValidationUtil.ts [hunk 1]
```diff
@@ -34,6 +34,10 @@ export function isValidEmail(email: string): boolean {
   return regExp.test(email);
 }
 
+export function isValidHandle(handle: string): boolean {
+  return /^@?[a-z_0-9.-]{2,256}$/.test(handle);
```
Comment (ffflorian) [inline packages/commons/src/main/util/ValidationUtil.ts:38]: This will validate handles like `bennyn` and `@bennyn`. I think we should stick to one.

Comment (bennyn) [reply packages/commons/src/main/util/ValidationUtil.ts:38]: I copied this one over from webapp.

Comment (ffflorian) [reply packages/commons/src/main/util/ValidationUtil.ts:38]: Ah, we then cut the leading `@`. Okay.

**Comment by bennyn** [comment]
Obsolete with https://github.com/wireapp/wire-web-packages/pull/3160

**Pull request closed without merging**